src/pages/Detail/DetailMain/DetailMain.js

```diff
@@ -0,0 +1,115 @@
+import React, { Component } from 'react';
+import DetailMainCommentWithoutValue from './Components/DetailMainComment/DetailMainCommentWithoutValue/DetailMainCommentWithoutValue';
+import DetailMainCommentWithValue from './Components/DetailMainComment/DetailMainCommentWithValue/DetailMainCommentWithValue';
+import DetailMainCommentModal from './Components/DetailMainComment/DetailMainCommentModal/DetailMainCommentModal';
+import DeleteCommentCheckModal from './Components/DetailMainComment/DeleteCommentCheckModal/DeleteCommentCheckModal';
+import DetailMainCommentEditModal from './Components/DetailMainComment/DetailMainCommentEditModal/DetailMainCommentEditModal';
+import DetailMainAct from './Components/DetailMainAct/DetailMainAct';
+import DetailMainInfor from './Components/DetailMainInfor/DetailMainInfor';
+import './DetailMain.scss';
+
+class DetailMain extends React.Component {
+  state = {
+    comment: '',
+    isCommentValue: false,
+    isCommentClicked: false,
+    isCheckCommentDelete: false,
+    isCommentEdited: false,
+  };
+
+  handleLeaveComment = () => {
+    this.setState({ isCommentClicked: true });
+  };
+
+  handleCloseComment = () => {
+    this.setState({ isCommentClicked: false });
+  };
```

Comment:
해당 함수는 하나로 합쳐 줄 수 있겠네요.
```suggestion
  handleCloseComment = () => {
    this.setState({ isCommentClicked: !isCommentClicked });
  };
```

Reply:
합쳤습니다.